irods/files/rule-bases/ipc-housekeeping.re

```diff
@@ -130,16 +130,54 @@ ipc_rescheduleStorageFreeSpaceDetermination {
 
 _ipc_rmTrash {
   writeLine('serverLog', 'DS: starting trash removal');
+  *zone = ipc_ZONE;
+  *verdict = true;
+  msiGetSystemTime(*timestamp, "");
+
+  # 2592000 is the epoch time for 30 days, we want to delete items which are older than 30 days in trash from the current date.
```

Comment:
Consider saying something like, "2,592,000 is the number of seconds in 30 days, ...". "Epoch" is an instant in time, not a duration.

Reply:
Thank you, done.